app/models/transaction.rb

```diff
@@ -8,6 +8,7 @@ class Transaction < ApplicationRecord
 
   scope :balance_transactions, -> { where(transactinable_type: 'BalanceTransaction') }
   scope :category_transactions, -> { where(transactinable_type: 'CategoryTransaction') }
+  scope :last_ten, -> { includes(:transactinable).limit(10).order('id desc') }
```

Comment:
Isn't it better to use `order(id: :desc)` instead of string?

Reply:
@sinpai `order(id: :desc)` only sorts, I want take 10 records

Reply:
@marderer can't you go with:
```ruby
includes(:transactinable).order('id desc').limit(10)
```
?